client/src/pages/certification.css

```diff
@@ -216,3 +216,20 @@
     margin-top: 20px;
   }
 }
+
+.certificate-outer-wrapper {
+  margin-top: calc(-1 * var(--header-height));
+  background-color: #fff;
+}
+
+.certificate-outer-wrapper .row,
```

Comment:
```suggestion
```
The rows should be left as `--gray-05`.  It's incredibly subtle, but it should look like
![image](https://user-images.githubusercontent.com/15801806/70847823-42d6e380-1e69-11ea-96e0-38283ff9598f.png)
rather than pure white.